tests/nightly/test_large_vector.py

```diff
@@ -708,6 +709,57 @@ def test_full():
     assert a[-1] == 3
 
 
+def test_load_save():
+    x = create_vector(size=LARGE_X)
+    nd.save('large_vector', [x])
+    y = nd.load('large_vector')
+    y = y[0]
+    assert x[0] == y[0]
+    assert x[-1] == y[-1]
+    os.remove('large_vector')
+
+
+def test_add_n():
+    x = [nd.ones(LARGE_X)]
+    y = nd.add_n(*x)
+    assert y[0] == 1
+    assert y[-1] == 1
+
+
+def test_modulo():
+    x = mx.nd.ones(LARGE_X)*6
+    y = mx.nd.ones(LARGE_X)*4
+    z = (x%y)
```

Comment:
why are brackets needed?

Reply:
readability ... it works fine without it too